pkg/monitor/cluster/clustersync.go

```diff
@@ -0,0 +1,67 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (mon *Monitor) emitClusterSync(ctx context.Context) error {
+	clusterSync, err := mon.hiveClusterManager.GetClusterSync(ctx, mon.doc)
+	if err != nil {
+		return err
+	}
+	if clusterSync != nil {
+		if clusterSync.Status.SyncSets != nil {
+			for _, s := range clusterSync.Status.SyncSets {
+				mon.emitGauge("hive.clustersync", 1, map[string]string{
+					"metric": "SyncSets",
```

Comment:
nit: having a dimension on the metric named `"metric"` might be a little confusing - should we rename this to something else? for example `syncType`?

Reply:
Done